packages/components/src/dropdown-menu/v2/README.md

```diff
@@ -0,0 +1,388 @@
+# `DropdownMenu` (v2)
+
```

Comment:
Should we add a callout?

```
<div class="callout callout-alert">
This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
</div>
```

Reply:
Yup, done in https://github.com/WordPress/gutenberg/pull/49473/commits/ddc1aa8623a37eb7b2ba042048257db121220ff2